k-distribution/include/kframework/builtin/domains.md

```diff
@@ -254,7 +254,7 @@ An element of a `Map` is constructed via the `|->` operator. The key is on the
 left and the value is on the right.
 
 ```k
-  syntax Map ::= KItem "|->" KItem                      [function, functional, hook(MAP.element), klabel(_|->_), symbol, latex({#1}\mapsto{#2})]
+  syntax Map ::= KItem "|->" KItem                      [function, functional, hook(MAP.element), klabel(_|->_), symbol, latex({#1}\mapsto{#2}), injective]
```

Comment:
Huh, why is this a `function` instead of a constructor?

Reply:
I mean, it's not really a regular constructor symbol in that the LLVM backend evaluates it to a built-in map with one element. However, the haskell backend, with the new unification algorithm, does need to understand that unifying `K1 |-> V1 ?= K2 |-> V2` is the same as unifying `K1 ?= K2 /\ V1 ?= V2`